dist/lib/PayIDValidator.php

```diff
@@ -5,11 +5,18 @@ class PayIDValidator {
     /**
      * Supported networks
      */
-    const NETWORK_ACH = 'ach';
     const NETWORK_ALL = 'all';
+
+    const NETWORK_ACH = 'ach';
```

Comment:
I saw these were alphabetical, but I think `all` is a special value different from an actual network, so I put it first.

Reply:
Works for me.